tests/cbs-xtpl-nbody/input.dat

```diff
@@ -20,26 +20,32 @@ set {
 }
 
 def unaug_focal(func, label, **kwargs):
-    kwargs['scf_basis'] = kwargs.get('scf_basis', 'cc-pVTZ')
-    kwargs['scf_scheme'] = kwargs.get('scf_scheme', driver_cbs.xtpl_highest_1)
 
-    kwargs['corl_wfn'] = kwargs.get('corl_wfn', 'mp2')
-    kwargs['corl_basis'] = kwargs.get('corl_basis', 'cc-pV[DT]Z')
-    kwargs['corl_scheme'] = kwargs.get('corl_scheme', driver_cbs.corl_xtpl_helgaker_2)
-
-    kwargs['delta_wfn'] = kwargs.get('delta_wfn', 'ccsd(t)')
-    kwargs['delta_wfn_lesser'] = kwargs.get('delta_wfn_lesser', 'mp2')
-    kwargs['delta_basis'] = kwargs.get('delta_basis', 'cc-pVDZ')
-    kwargs['delta_scheme'] = kwargs.get('delta_scheme', driver_cbs.xtpl_highest_1)
-
-    if label == 'custom_function':
-        label = 'Unaug Focal'
+    scf = {
+        'wfn': 'hf',
+        'basis': kwargs.pop('scf_basis', 'cc-pVTZ'),
+        'scheme': kwargs.pop('scf_scheme', 'xtpl_highest_1')
+    }
+    corl = {
+        'wfn': kwargs.pop('corl_wfn', 'mp2'),
+        'basis': kwargs.pop('corl_basis', 'cc-pV[DT]Z'),
+        'scheme': kwargs.pop('corl_scheme', 'corl_xtpl_helgaker_2')
+    }
+    delta = {
+        'wfn': kwargs.pop('delta_wfn', 'ccsd(t)'),
+        'wfn_lesser': kwargs.pop('delta_wfn_lesser', 'mp2'),
+        'basis': kwargs.pop('delta_basis', 'cc-pVDZ'),
+        'scheme': kwargs.pop('delta_scheme', 'xtpl_highest_1')
+    }
+    kwargs["cbs_metadata"] = [scf, corl, delta]
     return driver_cbs.cbs(func, label, **kwargs)
 
+register_composite_function(unaug_focal)
```

Comment:
We need documentation on `register_composite_function` and the use of `cbs_metadata`.

Reply:
For the most part, the updatehelpers guide the use. but never hurts to docstring the fn. `cbs_metadata` is pretty thoroughly demonstrated in https://github.com/psi4/psi4/blob/master/tests/cbs-xtpl-dict/input.dat . I'll add that hint to docs.